extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -200,6 +200,11 @@ public void build(ApplicationInfoBuildItem applicationInfo,
                             resourceEntry.getValue().getBytes(StandardCharsets.UTF_8)));
         }
 
+        try {
+            Files.delete(root);
+        } catch (IOException ignored) {
+
```

Comment:
Maybe it would be nice to log.debug here

Reply:
Sure, fixed